packages/hash/shared/src/createProseMirrorState.ts

```diff
@@ -42,6 +42,12 @@ export const createProseMirrorState = ({
       );
       return true;
     },
+    // execCommand is flagged as depecrated but it seems that there isn't a viable alternative
+    // to call the undo and redo default browser actions (https://stackoverflow.com/a/70831583)
+    // After the collab rework this should be replaced with a proper implementation
+    "Mod-z": () => document.execCommand("undo"),
+    "Ctrl-y": () => document.execCommand("redo"),
```

Comment:
```suggestion
    "Mod-y": () => document.execCommand("redo"),
```

Reply:
Do we want to allow Cmd + y for the redo action in macOs?

Reply:
Probably yes: https://www.technipages.com/ms-word-undo-redo

> Redo Keyboard Shortcut

(Although I don’t have MS Word to check the article)